.goreleaser.yml

```diff
@@ -74,15 +74,27 @@ nfpms:
       rpm:
         contents:
           - src: LICENSE
-            dst: "/usr/share/doc/{{ .PackageName }}-{{ .Version }}/LICENSE.txt"
+            dst: "usr/share/doc/{{ .PackageName }}-{{ .Version }}/LICENSE.txt"
```

Comment:
This change does not appear to do anything; the files in the RPM are still created with leading slashes, which you can check by running
```
rpm2cpio dist/pelican-7.1.1-next-1.arm64.rpm | cpio -t
```
for the pelican RPM the files start with `/`. You can compare that with an rpm you download via `yumdownloader` (so `yumdownloader wget` for example) and see that those have files that start with `./`.

I've also tried using `./` as the prefix (so `./usr/share/doc/...`) but that has the same results too.  So I think it's just another GoReleaser/NFPM limitation.

Reply:
Okay, I can change it back then. Tim T's rpmlint was yelling at him because we were using absolute paths instead of relative but that could have just been something weird on his end, I'm not super sure

Reply:
Maybe it was complaining about absolute paths as symlink targets? Those aren't allowed; fortunately that is something you can change - see https://github.com/htcondor/osdf-client/pull/86/files. (Pelican doesn't have symlinks yet but this is something to keep in mind for later since nFPM's example (https://goreleaser.com/customization/nfpm/) does it wrong.)

Reply:
So should I not worry about it and keep the leading /'s since we don't have symlinks yet?

Reply:
yep